curriculum/challenges/english/10-coding-interview-prep/data-structures/perform-a-union-on-two-sets.md

```diff
@@ -66,7 +66,7 @@ class Set {
   }
   // This method will return all the values in the set
   values() {
-    return Object.keys(this.dictionary);
+    return Object.values(this.dictionary);
   }
   // This method will add an element to the set
   add(element) {
```

Comment:
The `add` method in the challenge seed code will need to be updated to reflect the change as well.

Currently line 74 is still `this.dictionary[element] = true;`.

Reply:
Thanks for pointint that out, now I fixed it